data_ingestion/02_Ingest_data_with_Athena_v1.ipynb

```diff
@@ -0,0 +1,1144 @@
+{
```

Comment:
<p>Typo, and avoid using words like "simply" and "easy"</p><blockquote>To get started, you <s>simply</s> <strong>pointing</strong> to your data in Amazon S3,</blockquote><p>Should say:</p><blockquote>To get started, you point to your data in Amazon S3,</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>